### ISSUE TITLE
What is the learning rate to pre-train the DAMSM + GPT-2 model ?

### ISSUE THREAD

**Issue by ghost** [sidward14/Style-AttnGAN#31]
The configuration file to pretrain the DAMSM + GPT-2 model: https://github.com/sidward14/Style-AttnGAN/blob/8aa8da6601725c425e272f0bededa32033a38d84/code/cfg/DAMSM/bird.yml#L20

In your comment you mentioned to use a learning rate of 0.00005 for GPT-2. Should I change to 0.00005?

Thanks

**Comment by sidward14** [comment]
Yes, if I recall correctly, that is what I used. That looks to be a good starting place at the very least, since GPT-2 is much bigger.
